src/user.rs

```diff
@@ -0,0 +1,31 @@
+//! This module manages project users
+
+use std::path::Path;
+
+use Error;
+use utils;
+
+
+/// Creates a new user for the project in the current directory.
+/// Assumes the current directory contains a Protonfile.json file.
+///
+/// 1. Read the new user's public key from the file path given
+/// 2. Add the user's name and public key to the protonfile
+///
+/// Impure.
+pub fn new_user<P: AsRef<Path>>(public_key_path: P, name: String) -> Result<(), Error> {
+
+    // Get public key from file
```

Comment:
And see how readable this code is now? You don't even need these internal comments anymore.